### PR TITLE
Host module

### PR DESCRIPTION
**Pull request by adfaure**
Hi! 
I added the host module for remote simgrid. 
I implemented : 
 * by_name(string name)
 * current() 
 * speed()

Also the test which test the host module can be found at "tests/hostname".

### CODE REVIEW

File: include/rsg/host.hpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    	Host(const char *name, unsigned long int remoteAddr);
+      Host(const std::string name, unsigned long int remoteAddr);
+
+    public:
+      simgrid::xbt::string const& name() const { return name_; }
+      double speed();
+      ~Host();
+
+    public:
+      static Host *by_name(std::string name);
```
Comment (mquinson) [inline include/rsg/host.hpp]: It would be better if your by_name() function could return a reference to the object instead of a pointer.
We discussed about it for the Actor module, and I changed the S4U interface to return references. I was hopping that you guys could do the same change in the RSG world, please.

File: proto/rsg.proto [hunk 2]
```diff
@@ -65,6 +70,14 @@ message Request {
   message MBCreateRequest { required string name = 1; }
   optional MBCreateRequest mbCreate = 5;
 
+  //
+  //// Host
+  //
+  message HOSTGetByNAme { required string name = 1; }
+  optional HOSTGetByNAme hostGetByName = 6;
```
Comment (mquinson) [inline proto/rsg.proto]: typo?

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -16,6 +16,7 @@ set(CLIENT_SOURCES
     ${COMMON_SOURCES}
     client/rsg_actor.cpp   ../include/rsg/actor.hpp
     client/rsg_engine.cpp  ../include/rsg/engine.hpp
+    client/rsg_host.cpp ../include/rsg/host.hpp
```
Comment (mquinson) [inline src/CMakeLists.txt]: pliz keep it indented

File: tests/resources/two_hosts_platform.xml [hunk 4]
```diff
@@ -2,8 +2,9 @@
 <!DOCTYPE platform SYSTEM "http://simgrid.gforge.inria.fr/simgrid/simgrid.dtd">
 <platform version="4">
   <AS  id="AS0"  routing="Full">
-    <host id="host0" core="4" speed="8095000000f"/>
+    <host id="host0" core="4" speed="8095099999f"/>
```
Comment (mquinson) [inline tests/resources/two_hosts_platform.xml]: Why did you change that? I don't see the point, beside changing unrelated tesh files...

File: include/rsg/host.hpp (hunk 1 above)
Comment (adfaure) [inline include/rsg/host.hpp]: Ok, I will do the modification.

File: proto/rsg.proto (hunk 2 above)
Comment (adfaure) [inline proto/rsg.proto]: yes, thanks !

File: tests/resources/two_hosts_platform.xml (hunk 4 above)
Comment (adfaure) [inline tests/resources/two_hosts_platform.xml]: I wanted different values of speed. I can restor it if you wish.

Comment (mquinson) [inline tests/resources/two_hosts_platform.xml]: I see the point of having different values, but I'd prefer to have "nice values" that give non-decimal timings in the simulation. Yes, that's picky...
